linera-core/src/chain_worker/state/mod.rs

```diff
@@ -347,9 +347,12 @@ where
         Ok(())
     }
 
-    /// Returns the blobs requested by their `blob_ids` that are either in the chain manager's
-    /// pending blobs or in storage.
-    async fn get_blobs(&self, blob_ids: &HashSet<BlobId>) -> Result<Vec<Blob>, WorkerError> {
+    /// Returns the blobs requested by their `blob_ids` that are in the chain manager's pending blobs
+    /// and checks that they are otherwise in storage.
+    async fn get_blobs_and_checks_storage(
```

Comment:
I also don't think this renaming is necessary or makes sense given what the function does. If you wanna name everything we check in the function name, a proper name would be `get_blobs_from_chain_manager_and_check_storage`, but that's also not a great name IMHO 😅
I think `get_blobs` is perfectly fine here